blogposts/2021/api-documentation-generated-for-all-your-code.md

```diff
@@ -0,0 +1,84 @@
+---
+title: "Sneak peek: API documentation generated for all your code"
+description: "Here's a sneak peek of our latest experiment to bring API documentation to all your code, generated by LSIF code intelligence data."
+author: Stephen Gutekanst
+authorUrl: 
+publishDate: 2021-07-21T10:00-07:00
+tags: [blog, code, search, software, engineering, testing]
+slug: api-documentation-for-all-your-code
+heroImage: https://sourcegraphstatic.com/blog/api-docs/api-docs.png
+socialImage: https://sourcegraphstatic.com/blog/api-docs/api-docs.png 
+published: true
+---
+
+At Sourcegraph, we have a small team working hard on a new experimental feature: API docs.
+
+Using [LSIF code intelligence data](https://lsif.dev), we can analyze your code and generate API documentation for all of it. We've started with Go code and intend to expand to more languages as we develop this feature. 
```

Comment:
I think it's worth including a very brief description of Sourcegraph here. I try to imagine someone landing on this page from HN with no idea who we are or what we do, and think of what context would be helpful before diving in. 

I can take a stab, but I'd prefer an intro that connects the dots from what Sourcegraph does/offers to API docs ("we do X at Sourcegraph which gives us insight into Y, therefore we can generate API docs for you based on Y" – this is bad but I think you get the idea??)